src/includes/control/class-boldgrid-framework-control-col-width.php

```diff
@@ -115,7 +115,7 @@ public static function slider_device_group( $device, $current_layout ) {
 			if ( ! array_key_exists( 'items', $row ) ) {
 				continue;
 			}
-			$items_in_row      = count( $row['items'] );
+			$items_in_row      = count( $row['items'] ) ? count( $row['items'] ) : 1;
```

Comment:
I am wondering why a ternary was added/what caused the original count not to work. I am assuming if the $row['items'] is 0 it will set $items_in_row to 1 so that you are not dividing by 0?